dora/core/server/master/src/main/java/alluxio/master/job/DoraLoadJob.java

```diff
@@ -196,6 +195,7 @@ public DoraLoadJob(
    */
   public List<DoraLoadTask> prepareNextTasks() {
     LOG.debug("Preparing next set of tasks for jobId:{}", mJobId);
+    int workerNum = Scheduler.getInstance().getActiveWorkers().size();
```

Comment:
nit: can get from num of workers from getNextTasks, not need to call scheduler again

Reply:
done